facebook-commerce-events-tracker.php

```diff
@@ -87,14 +130,20 @@ public function __construct( $user_info, $aam_settings ) {
 			add_action( 'woocommerce_thankyou',                   [ $this, 'inject_purchase_event' ], 40 );
 
 			// TODO move this in some 3rd party plugin integrations handler at some point {FN 2020-03-20}
-			add_action( 'wpcf7_contact_form', [ $this, 'inject_lead_event_hook' ], self::FB_PRIORITY_LOW );
+			add_action( 'wpcf7_contact_form', [ $this, 'inject_lead_event_hook' ], 2 );
```

Comment:
Just out of curiosity, why did you change this priority?

Reply:
I did not; it's the same as the constant that was set there - it wasn't a good approach to use constants for setting these priorities as they're not universal; we moved away from those as we started maintaining the plugin, this was still lingering. Since the code block had some changes, I just swapped the constant for a plain number.

Reply:
@unfulvio , `FB_PRIORITY_LOW` was 11, from the PR diff. `FB_PRIORITY_HIGH` was 2... Just want to make sure we are not missing something here...

Reply:
you are very much right @deivamagalhaes ! my bad swapped the two, restored the original value of `11` here: https://github.com/facebookincubator/facebook-for-woocommerce/pull/1663/commits/24dcdcc2d674a019deb3818c5d307218e37d453c